libs/wasm-loader/Cargo.toml

```
@@ -11,7 +11,7 @@ version = "0.17.0"
 
 [dependencies]
 hex = "0.4"
-jsonrpsee = {version = "0.9", git = "https://github.com/paritytech/jsonrpsee", features = ["full"]}
+jsonrpsee = {version = "0.10.1", git = "https://github.com/paritytech/jsonrpsee", tag = "v0.10.1", features = ["full"]}
```

reduce the dependency tree a bit because the servers are not used :)

```suggestion
jsonrpsee = {version = "0.10.1", git = "https://github.com/paritytech/jsonrpsee", tag = "v0.10.1", features = ["client"]}
```

---

I will merge the PR as is and switch the `client`.